code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -0,0 +1,235 @@
+#define NULLROD "null rod"
+#define GODHAND "god hand"
+#define REDSTAFF "red staff"
+#define BLUESTAFF "blue staff"
+#define CLAYMORE "claymore"
+#define DARKBLADE "dark blade"
+#define SORD "sord"
+#define SCYTHE "scythe"
+#define CHAINSAW "chainsaw hand"
+#define CLOWNDAGGER "clown dagger"
+#define WHIP "whip"
+#define FEDORA "athiests fedora"
+#define ARMBLADE "dark blessing"
+#define CARP "carp-sie plushie"
+
+
+/obj/item/weapon/nullrod
+	name = "null rod"
+	desc = "A rod of pure obsidian, its very presence disrupts and dampens the powers of Nar-Sie's followers."
+	icon_state = "nullrod"
+	item_state = "nullrod"
+	force = 15
+	throw_speed = 3
+	throw_range = 4
+	throwforce = 10
+	w_class = 1
+	var/reskinned = FALSE
+
+/obj/item/weapon/nullrod/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is impaling \himself with the [src.name]! It looks like \he's trying to commit suicide.</span>")
+	return (BRUTELOSS|FIRELOSS)
+
+/obj/item/weapon/nullrod/attack_self(mob/user)
+	if(reskinned)
+		return
+	if(user.mind && (user.mind.assigned_role == "Chaplain"))
+		reskin_holy_weapon(user)
+
+/obj/item/weapon/nullrod/proc/reskin_holy_weapon(mob/M)
+	var/choice = input(M,"What theme would you like for your holy weapon?","Holy Weapon Theme") as null|anything in list(NULLROD, GODHAND, REDSTAFF, BLUESTAFF, CLAYMORE, DARKBLADE, SORD, SCYTHE, CHAINSAW, CLOWNDAGGER, WHIP, FEDORA, ARMBLADE, CARP)
```

Comment:
Instead of using defines, you can generate this list dynamically. Something like this:
```
holy_weapon_list = typesof(/obj/item/weapon/nullrod)
display_names = list()
for(var/V in holy_weapon_list)
	var/atom/A = V
	display_names += initial(A.name)
var/choice = input(M,"What theme would you like for your holy weapon?","Holy Weapon Theme") as null|anything in display_names
if(!src || !choice || M.stat || !in_range(M, src) || M.restrained() || !M.canmove || reskinned)
	return
var/index = display_names.Find(choice)
var/obj/item/weapon/nullrod/holy_weapon = holy_weapon_list[index]
```
That way they will be sorted by name. The only caveat is that you'll have to make sure that the name of these items is always unique.